src/DAL/repositories/jobRepository.ts

```diff
@@ -80,30 +80,44 @@ export class JobRepository extends GeneralRepository<JobEntity> {
       domain: req.domain,
     };
 
-    if (req.fromDate != undefined && req.tillDate != undefined) {
+    if (req.fromDate !== undefined && req.tillDate != undefined) {
```

Comment:
consider remove this 'undefined' condition

Reply:
cant, it is required bu the linter. if this condition is not set, the linter gives the following warning:  `warning  Unexpected nullable string value in conditional. Please handle the nullish/empty cases explicitly ` .
more over, this condition : `req.fromDate !== undefined` returns boolean while this req.fromDate returns undefined